lib/src/radio-group/radio-group.component.ts

```diff
@@ -0,0 +1,173 @@
+import { Component, Input, forwardRef, Provider, ViewChildren, QueryList, ElementRef, AfterViewChecked } from "@angular/core";
+import { NG_VALUE_ACCESSOR, ControlValueAccessor } from "@angular/forms";
+
+export interface RadioGroupItem {
+    /** The label or text to be displayed in the list */
+    label?: string;
+    /**
+     * A Custom template as html string to be used instead of label and description.
+     *
+     * Example: "<b>Hello</b>"
+     * */
+    customTemplate?: string;
```

Comment:
This can be a bit hard to understand, but i couldn't find a better way to do it, the only way i know is to declare this one as `TemplateRef<HTMLElement>` and then value you pass to it could be` "<span>test<span>" as unknown as HTMLElement`